modules/tinymce/src/plugins/table/main/ts/actions/TableActions.ts

```diff
@@ -101,6 +112,63 @@ export const TableActions = (editor: Editor, lazyWire: () => ResizeWire): TableA
 
   const pasteCells = execute(TableOperations.pasteCells, Fun.always, Fun.noop, lazyWire);
 
+  const setTableCellType = (editor: Editor, args: Record<string, any>) =>
```

Comment:
I'm not sure I like having all this logic here. However until we move more to snooker, I don't think it can be moved currently.

Reply:
Agreed